### PR TITLE
Add async load function for async loading of page details

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
For https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/324

File: mtp_common/assets-src/javascripts/modules/async-load.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  load: function (i, elem) {
+    var $elem = $(elem);
+    $.ajax({
+      'url': $elem.data('path'),
+      'method': 'GET',
+      'dataType': 'json'
+    }).success(function (data) {
+      var loaded_value = data[$elem.data('key')];
+      if (loaded_value != undefined) {
```
Comment (ushkarev) [inline mtp_common/assets-src/javascripts/modules/async-load.js]: `if (loaded_value)` would be simpler, but doesn't really matter! i'm cool with this. tho normally i think you do `typeof obj != "undefined"`

File: mtp_common/assets-src/javascripts/modules/async-load.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        $elem.html(loaded_value);
+      }
+    });
```
Comment (ushkarev) [inline mtp_common/assets-src/javascripts/modules/async-load.js]: do you think it'd be a good idea to have a `.error` fallback? e.g. a replacement piece of text or a callback function? not too bothered for this version tho, it's only for cell locations

**Pull request merged**